domains/crystalcoding.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "crystalcoding9999",
+           "email": "crystalcraft9999@gmail.com",
+           "discord": "1102272783712522331"
+        },
+    
+        "record": {
+            "CNAME": "crystalcoding9999.github.io"
```

Comment:
I can access your website but after a while i get redirected to https://crystalcoding.duckdns.org/ which returns an error caused by too many redirects

Reply:
pushed the fix